docs/reference/builder.md

```diff
@@ -297,9 +297,9 @@ Results in:
      ---> Running in a2c157f842f5
      Volume in drive C has no label.
      Volume Serial Number is 7E6D-E0F7
-    
+
```

Comment:
Be careful with your editor settings; it looks like it's stripping whitespace, which can quickly lead to too many unrelated changes and (in some case) trailing whitespace is added on purpose.

In this case I think it's ok to remove, but try to configure your editor to not automatically strip it (or make sure to revert those changes before committing)